exporter/exporterhelper/queued_retry_experimental_test.go

```diff
@@ -80,21 +80,21 @@ func TestGetRetrySettings(t *testing.T) {
 			expectedError: errNoStorageClient,
 		},
 		{
-			desc:           "obtain default storage extension",
-			numStorages:    1,
-			storageEnabled: true,
-			expectedError:  nil,
+			desc:          "obtain storage extension by name",
+			numStorages:   2,
+			storageID:     "1",
```

Comment:
Shouldn't this be?
```suggestion
			storageID:     "file_storage/1",
```
I thought ComponentID always includes the component type name.

Reply:
This is just a test parameter, in the actual test code we manually make it into a config.ComponentID struct. I'll change it to `storageName` so it's less confusing.